mission_control/navigator_launch/launch/motor_controller.launch

```diff
@@ -15,6 +15,9 @@
     <param name="port" value="/dev/serial/by-id/usb-FTDI_Roboteq_MDC1460_stbd-bow-if00-port0"/>
     <param name="value" value="115200" />
   </node>
+  <node pkg="rosserial_python" type="serial_node.py" name="arduino_serial">
```

Comment:
Can you make a new `hw_drivers.launch` and put this in there? We will put the hydrophones launch in there and any other hardware interfaces (excluding the motor drivers).